contracts/protocol/PositionManager.sol

```diff
@@ -29,16 +29,16 @@ contract PositionManager is
         bool isBuy,
         uint256 indexed amount,
         uint128 toPip,
-        uint256 passedPipcount,
-        uint128 partialFilledQuantity
+        uint256 passedPipCount,
+        uint128 remainingLiquidity
```

Comment:
in the formula, this function is calculated by totalLiquidityOfPip - remainingSizeOfOrder so it is equal to remaining liquidity that was not filled in that pip